### ISSUE TITLE
[webui] WebUI dependencies should be documented.

### ISSUE THREAD

**Issue by robertnishihara** [ray-project/ray#904]
The UI depends on a number of Python modules, e.g., it requires at least `pip install bokeh`. The full list should be documented somewhere.

cc @alanamarzoev @Wapaul1

**Comment by robertnishihara** [comment]
Also, to get the ipython widgets working, I had to run the following.

```
jupyter nbextension enable --py --sys-prefix widgetsnbextension
```

The error message I was seeing before just said

```
Widget Javascript not detected. It may not be installed properly.
```

**Comment by robertnishihara** [comment]
Fixed by #983.
